src/widget/form/settings/generalform.cpp

```diff
@@ -313,6 +314,11 @@ void GeneralForm::onReconnectClicked()
 void GeneralForm::reloadSmiles()
 {
     QList<QStringList> emoticons = SmileyPack::getInstance().getEmoticons();
+    if (emoticons.isEmpty())
+    // sometimes there are no emoticons available, don't crash in this case
+    {
+        qDebug() << "Settings GeneralForm: No emoticons found";
+    } else {
```

Comment:
If we're gonna be picky, this isn't Allman style which qTox largely adheres to, and also the rest should be indented.

It may be better (from a style standpoint) to just return after the qDebug, and not put the rest of the function inside the conditional.